### ISSUE TITLE
Bug fix release for Contao 4.4

### ISSUE THREAD

**Issue by netzarbeiter** [contao/contao#3302]
The bugs introduced with the security release 4.4.56 should also be fixed in Contao 4.4.

@ausi suggest waiting a week or two to see if any other problems occur.

**Comment by ausi** [comment]
The following fixes need to be backported to 4.4 I think?

- [x] #3291 #3332
- [x] #3292 #3328
- [x] #3293 #3331
- [x] #3314 #3330
- [x] #3315 #3327
- [x] #3319 #3333

**Comment by MacKP** [comment]
#3291 must be backported.
I have a Contao 4.4 with that:
```
<a href="{{iflng::en}}{{link_url::196}}{{iflng::en-US}}{{link_url::4723}}{{iflng::de}}{{link_url::3744}}{{iflng}}" title="xxx">
```
The output is now:
```
<a href="" title="xxx">
```
Kind regards
